modules/adagioBidAdapter.js

```diff
@@ -194,7 +195,7 @@ function _buildVideoBidRequest(bidRequest) {
   }
 
   bidRequest.mediaTypes.video = videoParams;
-  validateOrtbVideoFields(bidRequest);
+  validateOrtbFields(bidRequest, 'video');
```

Comment:
out of scope for this PR, but this is unnecessary - https://github.com/prebid/Prebid.js/issues/13049